markov.py

```diff
@@ -34,7 +35,9 @@ def get_model(chat):
     chat_id = str(chat.id)
     chat_messages = db.find_one(chat_id=chat_id)
     if chat_messages:
-        return markovify.text.NewlineText(chat_messages['text'])
+        text = chat_messages['text']
+        text_limited = "\n".join(text.splitlines()[0:MESSAGE_LIMIT])
```

Comment:
I think this line could turn into a new method, and then it could be easily tested